### ISSUE TITLE
whatsapp data extractor can't find temp folder

### ISSUE THREAD

**Issue by anmac1789** [EliteAndroidApps/WhatsApp-Key-DB-Extractor#55]
Hi everyone, I got this error:

![image](https://github.com/EliteAndroidApps/WhatsApp-Key-DB-Extractor/assets/103609429/585793a8-0169-45a8-b769-867a82245d6e)

**Comment by MadelynMiaw** [comment]
Me too

**Comment by anmac1789** [comment]
> Me too

are you having the same problem or similar problem ?

**Comment by MadelynMiaw** [comment]
Yes, I just solved it. The ADB driver is not installed correctly on my PC. I followed the steps in the YouTube video, and it worked for me 

https://m.youtube.com/watch?v=GERlhgCcoBc

**Comment by anmac1789** [comment]
> Yes, I just solved it. The ADB driver is not installed correctly on my PC. I followed the steps in the YouTube video, and it worked for me
> 
> https://m.youtube.com/watch?v=GERlhgCcoBc

I thought this is related to java

**Comment by MadelynMiaw** [comment]
Do you solve it?

**Comment by anmac1789** [comment]
> Do you solve it?

no it didn't work I already have adb installed and tried running the adb devices command it shows its connected but still...even then same error. I'm thinking it has something to do with java version

**Comment by anmac1789** [comment]
![image](https://github.com/EliteAndroidApps/WhatsApp-Key-DB-Extractor/assets/103609429/77dd2c84-a593-4cfa-bcb3-773591c81084)

This is still what I got...

WHAT THE HELL MAN...when I went back to Whatsapp, everything got wiped !!!!! WTF IS THIS SHITTY SOFTWARE !!!

**Comment by anmac1789** [comment]
please someone help !
